EIPS/eip-4973.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 4973
+title: Non-Transferrable Non-Fungible Tokens (Soulbound Tokens or "Badges")
+description: A standard interface for non-transferrable non-fungible tokens, also known as "soulbound tokens" (short "SBT") or "badges".
+author: Nicola Greco, Tim Daubenschütz (TimDaub)
+discussions-to: https://ethereum-magicians.org/t/eip-4973-non-transferrable-non-fungible-tokens-soulbound-tokens-or-badges/8825
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-04-01
+requires: 165, 721
+---
+
+## Abstract
+
+The following standard allows for implementing a standard API for SBTs within smart contracts. It provides basic functionality to gift, mint and track SBTs.
+
+## Motivation
+
+In various GitHub issues, pull requests, and Ethereum's governance forums, its community, has expressed a need for non-transferrable, non-fungible tokens. Sometimes those "NTTs" are referred to as "soulbound tokens" ("SBTs") in reference to popular MMORPGs such as World of Warcraft, where items, upon looting, are bound with the player's soul and hence could not subsequently be transferred to other characters anymore. Other times, when using them as credentials, NTTs have been called "badges."
+
+The purpose of this document is to make NTTs a reality on Ethereum by creating consensus around a **maximally backward-compatible** but otherwise **minimal** interface definition.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+`ERC4973` tokens must define the interface function of [`ERC165`](./eip-165.md) "Standard Interface Definition." An `ERC4973` must support the [`ERC721`](./eip-721.md)'s `ERC721Metadata` interface with the hexadecimal identifier `0x5b5e139f`. It must not, however, support [`ERC721`](./eip-721.md)'s tracking and transferring functionality with the interface identifier `0x80ac58cd`. Neither [`ERC721`](./eip-721.md)'s `ERC721TokenReceiver` with the interface identifier `0x150b7a02` must be supported.
```

Comment:
Hope you don't mind me rewording this a little:

```suggestion
`ERC-4973` tokens _must_ implement the interfaces:

 - [ERC-165](./eip-165.md)'s `ERC165` (`0x01ffc9a7`)
 - [ERC-721](./eip-721.md)'s `ERC721Metadata` (`0x5b5e139f`)
 - [ERC-721](./eip-721.md)'s `ERC721TokenReceiver` (`0x150b7a02`)

`ERC-4973` tokens _must not_ implement the interfaces:

 - [ERC-721](./eip-721.md)'s `ERC721` (`0x80ac58cd`)
```

I wasn't sure about `ERC721TokenReceiver`, so adjust as required.